source/includes/common-mc-admin-config.rst

```diff
@@ -1122,4 +1122,146 @@ MinIO returns an error if the Kakfa broker verison does not match those specifie
+
+Set to ``on`` to encrypt objects after compressing them.
+Defaults to ``off``.
+
+.. admonition:: Encrypting compressed objects may compromise security
+   :class: warning
+
+   MinIO strongly recommends against encrypting compressed objects.
+   If you require encryption, carefully evaluate the risk of potentially leaking information about the contents of encrypted objects.
+	   
+.. end-minio-data-compression-allow_encryption-desc
+
+.. start-minio-data-compression-comment-desc
+
+Specify a comment to associate with the data compression configuration.
+   
+.. end-minio-data-compression-comment-desc
+
+.. start-minio-data-compression-enable-desc
+
+Set to ``on`` to enable data compression for new objects.
+Defaults to ``off``.
+
+Enabling or disabling data compression does not change existing objects.
+
+.. end-minio-data-compression-enable-desc
+
+.. start-minio-data-compression-extensions-desc
+
+Comma-separated list of the file extensions to compress.
+Setting a new list of file extensions replaces the previously configured list.
+Defaults to ``.txt,.log,.csv,.json,.tar,.xml,.bin``.
+
+.. admonition:: Uncompressable files
+   :class: note
+
+   MinIO will *not* compress the following types of files, even if specified in an :mc-conf:`~compression.extensions` argument:
+
+   .. list-table:: Uncompressable File Extensions
+      
+      * - File Format
+	- File Extensions
+
+      * - GZIP
+	- ``.gz``
+	  
+      * - BZIP2
+	- ``.bz2``
+
+      * - WinRAR
+	- ``.rar``
+	
+      * - ZIP
+	- ``.zip``
+	
+      * - 7-Zip
+	- ``.7z``
+	  
+      * - Lempel–Ziv–Markov chain algorithm (LZMA)
+	- ``.xz``
+	  
+      * - MPEG-4 Part 14 (MP4)
+	- ``.mp4``, ``.m4a``, ``.m4p``, ``.m4b``, ``.m4r``, ``.m4v``
+
+      * - Matroska (MKV)
+	- ``.mkv``, ``mk3d``, ``.mka``
+
+      * - QuickTime File Format (QTFF)
+	- ``.mov``, ``.qt``
+     
+   These files cannot be significantly reduced in size, therefore MinIO will not attempt to compress them.
+
+.. end-minio-data-compression-extensions-desc
+
+.. start-minio-data-compression-mime_types-desc
+
+Comma-separated list of the MIME types to compress.
+Setting	a new list of types replaces the previously configured list.
+Defaults to ``text/*, application/json, application/xml, binary/octet-stream``.
+
+.. admonition:: Uncompressable files
+   :class: note
+
+   MinIO will *not* compress the following types of files, even if specified in a :mc-conf:`~compression.mime_types` argument:
+
+   .. list-table:: Uncompressable File Types
+
+      * - File format
+	- Media Type (MIME type)
+
+      * - Video
+	- ``video/*``
+
+      * - Audio
+	- ``audio/*``
+
+      * - ZIP application
+	- ``application/zip``
+
+      * - GZIP application
+	- ``application/x-gzip``
+
+      * - BZIP2 application
+	- ``application/x-bz2``
+
+      * - Lempel–Ziv–Welch (LZW) application
+	- ``application/x-compress``
+
+      * - LZMA application
+	- ``application/x-xz``
+
+
+   +------------------------+----------------------------+
```

Comment:
Table experiment two: grid tables

I haven't found a way to have multiple heading rows in the same table.

Reply:
Could do tabs and just have one table for file types and one for compressed files.